src/Ledger/Ratify.lagda

```diff
@@ -539,18 +545,26 @@ private variable
   removed : ℙ (GovActionID × GovActionState)
   d : Bool
 
-data _⊢_⇀⦇_,RATIFY'⦈_ : RatifyEnv → RatifyState → GovActionID × GovActionState → RatifyState → Type where
-
 \end{code}
 \begin{figure*}[ht]
 \begin{AgdaSuppressSpace}
 \begin{code}
-  RATIFY-Accept : ∀ {Γ} {es} {removed} {d} {a} {es'} → let open RatifyEnv Γ; st = a .proj₂; open GovActionState st in
+data _⊢_⇀⦇_,RATIFY'⦈_ :
+  RatifyEnv → RatifyState → GovActionID × GovActionState → RatifyState → Type where
+
+  RATIFY-Accept :  {Γ        : RatifyEnv}
+                   {es es'   : EnactState}
+                   {removed  : ℙ (GovActionID × GovActionState)}
+                   {d        : Bool}
+                   {a        : GovActionID × GovActionState}
+
+     → let open RatifyEnv Γ; st = a .proj₂; open GovActionState st in
+
      ∙ acceptConds Γ ⟦ es , removed , d ⟧ʳ a
      ∙ ⟦ a .proj₁ , treasury , currentEpoch ⟧ᵉ ⊢ es ⇀⦇ action ,ENACT⦈ es'
        ────────────────────────────────
-       Γ ⊢  ⟦ es   , removed          , d                      ⟧ʳ ⇀⦇ a ,RATIFY'⦈
-            ⟦ es'  , ❴ a ❵ ∪ removed  , delayingAction action  ⟧ʳ
+       Γ ⊢ ⟦ es  , removed         , d                     ⟧ʳ ⇀⦇ a ,RATIFY'⦈
+           ⟦ es' , ❴ a ❵ ∪ removed , delayingAction action ⟧ʳ
```

Comment:
No need for extra white space since we're not relying on `agda` to handle alignment here.